cmd/signozschemamigrator/schema_migrator/logs_migrations.go

```diff
@@ -80,4 +80,55 @@ ORDER BY name ASC`,
 			},
 		},
 	},
+	{
+		MigrationID: 1001,
+		UpItems: []Operation{
+			CreateTableOperation{
+				Database: "signoz_logs",
+				Table:    "tag_attributes_v2",
+				Columns: []Column{
+					{Name: "unix_milli", Type: ColumnTypeInt64, Codec: "Delta(8), ZSTD(1)"},
+					{Name: "tag_key", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+					{Name: "tag_type", Type: LowCardinalityColumnType{ColumnTypeString}, Codec: "ZSTD(1)"},
+					{Name: "tag_data_type", Type: LowCardinalityColumnType{ColumnTypeString}, Codec: "ZSTD(1)"},
+					{Name: "string_value", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+					{Name: "number_value", Type: NullableColumnType{ColumnTypeFloat64}, Codec: "ZSTD(1)"},
+				},
+				Indexes: []Index{
+					{Name: "string_value_index", Expression: "string_value", Type: "ngrambf_v1(4, 1024, 3, 0)", Granularity: 1},
+					{Name: "number_value_index", Expression: "number_value", Type: "minmax", Granularity: 1},
+				},
+				Engine: ReplacingMergeTree{
+					MergeTree: MergeTree{
+						PartitionBy: "toDate(unix_milli / 1000)",
+						OrderBy:     "(tag_key, tag_type, tag_data_type, string_value, number_value)",
+						TTL:         "toDateTime(unix_milli / 1000) + toIntervalSecond(1296000)",
```

Comment:
Also why 15 days ? will it be same as table TTL ?

Reply:
Because fast filters don't work if they don't exist for two days https://github.com/SigNoz/engineering-pod/issues/1756. It won't be the same as the main table TTL. I think 15 should be good for both those who send data constantly and those who send data occasionally.

Reply:
Ok